design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,211 @@
+type Mutation {
+    recalculateNegotiableQuotePrices(
+        quote_ids: [String]!
+    ): RecalculateNegotiableQuotePricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestNegotiableQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestNegotiableQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    setShippingMethodsOnNegotiableQuote(
+        negotiableQuoteShippingMethod: NegotiableQuoteShippingMethodInput!
+    ): setShippingMethodsOnNegotiableQuoteOutput @doc(description:"Set negotiable quote shipping method")
+}
+
+type Customer {
+    negotiable_quote (
+        id: ID!): NegotiableQuote @doc(description: "Get negotiable quote of a customer")
+
+    negotiable_quotes(
+        page_size: Int = 20,
+        current_page: Int = 1,
+        filter: NegotiableQuoteFilterInput
+    ): [NegotiableQuote] @doc(description: "Get negotiable quotes of a customer")
+
+    estimateShippingCostForNegotiableQuote(
+        address: NegotiableQuoteShipAddressInput!
+    ): [ShippingMethodsOutput]! @doc(description:"Estimate shipping costs for a negotiable quote")
+
+    estimateShippingCostByAddressIdForNegotiableQuote(
+        address_id: String!
+    ): [ShippingMethodsOutput]! @doc(description:"Estimate shipping costs for a negotiable quote by Address ID")
+
+    estimateShippingInformationForNegotiableQuote(
+        address_info: NegotiableQuoteAddressInformationInput!
+    ): NegotiableQuotePaymentInfo! @doc(description:"Get payment options and the order totals by address and shipping info")
+}
+
+type NegotiableQuote {
+    items: [NegotiableQuoteItem] @doc(description: "Negotiable quote item")
+    billing_address: BillingAddress @doc(description: "Billing address of negotiable quote")
```

Comment:
Please help me to understand in which scenario on the storefront negotiable quote billing address is visible to the customer?

Reply:
Removed the `billing_address` field. Thanks